Employee.java

```diff
@@ -0,0 +1,96 @@
+package home.work;
+
+public class Employee {
+
+    private String nameone;
+    private String nametwo;
+    private String email;
+    private String phonenumber;
+    private String position;
+    private int age;
+    private int the_salary;

```

Comment:
camelCase нотация должна использовать для именовании в Java всего, кроме констант
```suggestion
    private int salary;

```